### PR TITLE
update readme to include oauth connection information

### PR DESCRIPTION
**Pull request by aleachjr**
ping @lord2800 @bc-bfenton

### CODE REVIEW

**Comment by lord2800** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -74,6 +75,16 @@ Bigcommerce::configure(array(
 ));
 ```
 
+oAuth
+```
+Bigcommerce::configure(array(
+    'client_id' => 'xxxxxxxx',
+    'auth_token' => 'xxxxxxx',
+    'store_hash' => 'xxxxxxx'
+
```
Comment (bc-bfenton) [inline README.md]: extra blank line here

File: README.md [hunk 2]
```diff
@@ -74,6 +75,16 @@ Bigcommerce::configure(array(
 ));
 ```
 
+oAuth
```
Comment (bc-bfenton) [inline README.md]: typically spelled OAuth
I'd also make this and Basic Auth at least a subheading, formatting-wise, for easier skimming

File: README.md [hunk 3]
```diff
@@ -42,9 +42,9 @@ API client from [the Bigcommerce vendor on Packagist](https://packagist.org/pack
 You can also install composer for your specific project by running the following in the library folder.
 
 ```
```
Comment (bc-bfenton) [inline README.md]: might as well add syntax highlighting for "shell"

File: README.md [hunk 4]
```diff
@@ -82,68 +92,68 @@ the store, ping the getTime method which will return a DateTime object
 representing the current timestamp of the store if successful or false if
 unsuccessful:
 
-```
+~~~php
 $ping = Bigcommerce::getTime();
 
 if ($ping) echo $ping->format('H:i:s');
```
Comment (bc-bfenton) [inline README.md]: I personally hate braceless conditionals

File: README.md [hunk 5]
```diff
@@ -82,68 +92,68 @@ the store, ping the getTime method which will return a DateTime object
...
 Accessing collections and resources (GET)
 -----------------------------------------
 
 To list all the resources in a collection:
 
-```
+~~~php
 $products = Bigcommerce::getProducts();
 
 foreach($products as $product) {
```
Comment (bc-bfenton) [inline README.md]: space after foreach

**Comment by bc-bfenton** [comment]
:+1:

**Pull request merged**